src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -251,12 +262,15 @@
         {:endpoint (endpoints/get-entity-types (:workspace-id props))
          :on-done (fn [{:keys [success? get-parsed-response status-text]}]
                     (if success?
-                      (.add (:engine @locals)
-                            (clj->js (->> (get-parsed-response)
-                                          vals
-                                          (map :attributeNames)
-                                          flatten
-                                          (map (partial str "this.")))))
+                      (let [loaded-config (:loaded-config @state)
+                            entity-types (get-parsed-response)
+                            workspace-attributes (get-in props [:workspace :workspace :workspace-attributes])
+                            selected-entity-type (get-in loaded-config [:methodConfiguration :rootEntityType])]
+                        (refresh-autocomplete {:engine (:engine @locals)
+                                               :workspace-attributes workspace-attributes
+                                               :entity-types entity-types
+                                               :selected-entity-type selected-entity-type})
+                        (swap! state assoc :entity-types entity-types))
```

Comment:
I can't get that approach to work, but the advice does make sense. Will sit down with someone to go over it.